include/dos_system.h

```diff
@@ -55,7 +55,7 @@ class DOS_DTA;
 
 class DOS_File {
 public:
-	DOS_File():flags(0),time(0),date(0),attr(0),refCtr(0),open(false),name(0),hdrive(0xff) { };
+	DOS_File():flags(0),time(0),date(0),attr(0),refCtr(0),open(false),name(0),newtime(false),hdrive(0xff) { };
```

Comment:
Stylistic request; we are developing our style for formatting constructors, that is a bit more readable, format it this way, please:
https://github.com/dreamer/dosbox-staging/wiki/DOSBox-coding-style-(draft)#formatting-constructors

Reply:
Visual Studio auto-formatting currently does not support formatting member initializers this way. Are you sure you want to go through with this?

Reply:
Visual Studio auto-formatting will use clang-format rules once we'll have them finalized. As for now, keeping the whole initializer list in a single line is the worst possible formatting decision.

Reply:
Oh, I'm not arguing for single-line list but it *can* be auto-formatted if
you keep colon on the same line and put members on the second line and
beyond.

Reply:
This is a cross-platform project and none of the existing maintainers uses Visual Studio for normal development. We maintain up-to-date solution file to make the project inviting to new Windows developers and for easy CI access on GitHub Actions infrastructure.

clang-format will work cross-platform in all supported environments, without depending on IDE-specific quirks.

Reply:
As you wish, then.